apps/condo/domains/common/utils/locales.js

```diff
@@ -0,0 +1,13 @@
+
+// TODO(mrfoxpro): typescript enum is good here
+const defaultLocale = 'en'
+
+const extractReqLocale = (req) => {
+    const headersLocale = req.headers['accept-language'] && req.headers['accept-language'].slice(0, 2)
```

Comment:
we already have a cookie language checks. use it too

Reply:
@pahaz should we really check cookie languge?

Reply:
we already use this logic inside withIntl